sdks/python/apache_beam/testing/benchmarks/nexmark/models/bids_per_session.py

```diff
@@ -0,0 +1,56 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Result of Query11 and 12."""
+from __future__ import absolute_import
+
+from apache_beam.coders import coder_impl
+from apache_beam.coders.coders import FastCoder
+from apache_beam.testing.benchmarks.nexmark import nexmark_util
+
+
+class BidsPerSessionCoder(FastCoder):
```

Comment:
Does every result class need a coder? What are the coder used for?

Reply:
I'm actually not sure, we need coder for the 3 models. As for the results, IIRC according to the documentation, beam will try to infer coders if no coder is registered. I am not sure how exactly that works and if the inferred coder works well on all runners. 
So far I forgot to register coders for some of the result models and there seems to be no problem on the direct runner. not sure on others though.

Reply:
I think there will be further transforms, at least there is going to be metrics transform after the query is done to collect data and re-stamping the elements for streaming purposes i suppose

Reply:
can't we use tuple or dict instead? that way the we can infer the coders instead of defining additional ones.

Reply:
I think the most reason to use a wrapper is to have a clearer idea of what each field is, and it would provide a less error-prone way to construct than using dict. I think by using dict we would need to define literal name of the fields into variables anyway to prevent making typo errors.

Reply:
The pipeline is largely static, any typo issue should be resolved after the pipeline is validated once.